### PR TITLE
ci: adding action for generating install.yaml

### PR DESCRIPTION
**Pull request by devotional-phoenix-97**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [x] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- For every release we generate a install.yaml and upload it to the repo and the release

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/installyaml.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    strategy:
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install kustomize
+        run: make kustomize
+
+      - name: Generate install.yaml
+        run: make build-installer
```
Comment (devotional-phoenix-97) [inline .github/workflows/installyaml.yaml:22]: Add release as env var

**Comment by mvdkleijn** [comment]
Just checking... was this ready to merge as far as you're concerned @devotional-phoenix-97 ?

**Comment by devotional-phoenix-97** [comment]
> Just checking... was this ready to merge as far as you're concerned @devotional-phoenix-97 ?

It is.
We might need to issue more PR's to test and get this to a fully functional state.
But before we can, we need to merge this and issue a draft PR.
Can your approve?

**Review by hikarukin — APPROVED** [review]
LGTM

**Pull request merged**